Sources/PackageCollections/PackageCollections.swift

```diff
@@ -298,7 +307,7 @@ public struct PackageCollections: PackageCollectionsProtocol {
     }
 
     func findPackage(
-        identifier: PackageReference.PackageIdentity,
+        identifier: String,
```

Comment:
Another case that should probably not be replaced with `String`.

Reply:
Resolved by c3fbf4e08ddd54a24fd24b0900702b3e747ad65a